setup.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 IBM. All Rights Reserved.
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
You can drop this second copyright notice -- no Google IP here.

Reply:
Oh, ok - I copied it from one of the source files